ci/tests/puppeteer/scenarios/login-success-jetty/script.js

```diff
@@ -0,0 +1,17 @@
+const puppeteer = require('puppeteer');
```

Comment:
Rename the scenario name so jetty is the first word (similar to how tomcat tests are now)